.travis.yml

```diff
@@ -1,10 +1,23 @@
 language: python
-python:
-    - 2.7
-install:
-    - "pip install tox"
-env:
-    - TOX_ENV=django18
-    - TOX_ENV=django111
-script:
-    - tox -e $TOX_ENV
+matrix:
+  include:
+    - install:
+      - git clone https://github.com/edx/devstack.git
+      - (cd devstack; sed -i 's/:cached//g' ./docker-compose-host.yml)
```

Comment:
Can you include a comment explaining this and why it's necessary to replace ":cached" in the file?